### PR TITLE
:window: :art: New design and cleanup for CircleLoader

### PR DESCRIPTION
**Pull request by timroes**
## What
Improve the design of the StatusIcon circular loader:

https://user-images.githubusercontent.com/877229/193931234-f1939df7-b6e1-429d-984b-175b8724cc4e.mp4

https://user-images.githubusercontent.com/877229/193931926-25871be9-ab0b-40f2-8ff3-42d4bc62c16e.mp4

This also cleans up the CircleLoader component and move it from styled-component to SCSS.

**Note:** The CSS classes in the snapshots seemed to change just as part of compiling new SCSS. We should exclude them from being matched, but I think that should be separate from this PR.

### CODE REVIEW

File: airbyte-webapp/src/components/StatusIcon/CircleLoader.module.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  animation: spinning 1000ms linear infinite normal forwards;
+}
+
+.centerDot {
+  fill: colors.$blue;
+}
+
+:export {
+  // Export colors to be used in SVG gradients
+  gradientStart: colors.$blue;
```
Comment (timroes) [inline airbyte-webapp/src/components/StatusIcon/CircleLoader.module.scss:23]: :information_source: `linear-gradient` doesn't work for `fill` on SVG elements, thus the gradient still need to be defined in SVG. To get access to the colors in JS, I exported them from here explicitally. They are already exported from the `theme.ts` file, but I consider this still to be going away once we're removing styled components, so in this case where we'll need to have this export even with SASS, I wanted to make it more explicit.

File: airbyte-webapp/src/components/StatusIcon/CircleLoader.module.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  gradientStop: colors.$blue-transparent;
+}
```
Comment (josephkmh) [inline airbyte-webapp/src/components/StatusIcon/CircleLoader.module.scss:25]: Does these need to be re-named and re-exported here? Seems simpler to just import directly from `./scss/export.module.scss` in `CircleLoader.tsx`?

**Review by joeykmh — APPROVED** [review]
Just one minor comment, otherwise looks good to me 👍

**Review by edmundito — APPROVED** [review]
Tested locally and LGTM

File: airbyte-webapp/src/components/StatusIcon/CircleLoader.module.scss (hunk 2 above)
Comment (timroes) [reply airbyte-webapp/src/components/StatusIcon/CircleLoader.module.scss:25]: Hmm great my comment that I explained that in somehow vanished :thinking:

The idea is that `export.module.scss` is deprecated and meant to be removed onces we switched everything away from styled-components to SASS. This is a (very rare) place that will need though to still have access to those colors in TS even with styled-component removed, so I decided to excplicitally export them here, to not later block the removal of the export.module.scss.

Comment (josephkmh) [reply airbyte-webapp/src/components/StatusIcon/CircleLoader.module.scss:25]: I see, didn't realize that `export.module.scss` is deprecated along with `theme.ts` 👍

**Pull request merged**